fixes (#28)
ISSOtm/gb-asm-tutorial: src/part1/header.md

> The `ds` fills all bytes from $103 to $14F (inclusive) with the value $00.

Reviewer: ```suggestion
The directive `ds` stands for "define space", and allows filling a range of memory.
This specific line fills all bytes from $103 to $14F (inclusive) with the value $00.
```

Author: Great

Reviewer: Where are they mentioned?

Author: Tracing>setup at the screenshot

Reviewer: They're intentionally not explained yet, they'll be brought up somewhere during Part I or Part II.

Author: Alright, you did mention everything that would be explained in part 2. Just not this one.

Since different pieces of code and/or data cannot overlap, this ensures that the header's memory range can safely be overwritten by RGBFIX, and that nothing else accidentally gets steamrolled instead.

> It may not be obvious how `ds` is told to fill this specific memory range.

Reviewer: ```suggestion
It may not be obvious how this `ds` ends up filling that specific memory range.
```

Author: Either is fine, but I just recommend briefly mentioning what it stands for just so people don't have to think about what it means. Not necessarily to memorize it.
